health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/AppFeatures.js

```
@@ -101,49 +122,43 @@ const AppFeatures = () => {
-              </HeaderComponent>
-              <hr style={{ border: "1px solid #e0e0e0", width: "100%", margin: "0.5rem 0" }} />
-              <p style={{ margin: "0rem" }}>{t(feature.DESC)}</p>
-              <Button
-                className={"campaign-module-button"}
-                type={"button"}
-                size={"large"}
-                variation={selectedModuleCodes?.[code]?.includes(feature?.code) ? "secondary" : "primary"}
-                label={selectedModuleCodes?.[code]?.includes(feature?.code) ? t("DESELECT") : t("ES_CAMPAIGN_SELECT")}
-                onClick={() => handleSelectModule(feature?.code)}
+        {selectedModuleFeatures?.map((feature, index) => (
+          <Card className={`module-card ${selectedModuleCodes?.[code]?.includes(feature?.code) ? "selected-card" : ""}`}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)